clang/test/Frontend/fixed_point_bit_widths.c

```diff
@@ -1,7 +1,7 @@
 // RUN: %clang -x c -ffixed-point -S -emit-llvm -o - %s | FileCheck %s
-// RUN: %clang -x c -ffixed-point -S -emit-llvm -o - --target=x86_64-scei-ps4-ubuntu-fast %s | FileCheck %s
```

Comment:
@pogo59

Reply:
Thanks for tagging me. These are definitely invalid triples. It appears they were added in f921d854, which was a recommit of db55d833, where the test was introduced. I don't see that they are really adding anything to the coverage, so I think it's fine to remove them.